### DISCUSSION TITLE
important icons

### DISCUSSION THREAD

**Discussion by ahmetcanisik** [tailwindlabs/heroicons#1175] [Ideas]
important(!) icons please

**Comment by ralphjsmit** [comment]
These you can use perhaps?

<img width="1020" alt="Scherm­afbeelding 2024-05-14 om 13 57 03" src="https://github.com/tailwindlabs/heroicons/assets/59207045/818461b8-2d19-4fb5-930e-b7a4f86a2d96">

**Reply by ahmetcanisik** [reply]
I didn't know these, these are the icons that will make my job easier. Thank you.
